xbmc/utils/EGLFence.cpp

```diff
@@ -135,8 +135,7 @@ void CEGLFence::WaitSyncCPU()
 
   EGLint status{EGL_FALSE};
 
-  while (status != EGL_CONDITION_SATISFIED_KHR)
-    status = m_eglClientWaitSyncKHR(m_display, m_kmsFence, 0, EGL_FOREVER_KHR);
+  status = m_eglClientWaitSyncKHR(m_display, m_kmsFence, 0, EGL_FOREVER_KHR);
```

Comment:
Status variable is now useless. However, some kind of log would be useful if wait fails.

Reply:
Yes, true, return value is useless, but log in case of an error is pushed asynchronously in https://github.com/xbmc/xbmc/blob/f02afef0f39f8a3770d0e9fd5483e8a53a1814d7/xbmc/utils/EGLUtils.cpp#L96

Reply:
Did a fixup about that, also destroyed the sync on when snyc request is succesfull, there is no need to destroy when sync is unsuccesfull.

syncrequest can also return a timeout, but in our case this is impossible since the requested time is infinite so i think there is no need to handle this case.